Description: Let configure check detect version-less libdbs to support
 newer versions without patching the configure code.
Origin: vendor
Forwarded: no
Last-Update: 2010-01-18

--- php5.orig/ext/dba/config.m4
+++ php5/ext/dba/config.m4
@@ -362,7 +362,7 @@ if test "$PHP_DB4" != "no"; then
       break
     fi
   done
-  PHP_DBA_DB_CHECK(4, db-5.1 db-5.0 db-4.8 db-4.7 db-4.6 db-4.5 db-4.4 db-4.3 db-4.2 db-4.1 db-4.0 db-4 db4 db, [(void)db_create((DB**)0, (DB_ENV*)0, 0)])
+  PHP_DBA_DB_CHECK(4, db, [(void)db_create((DB**)0, (DB_ENV*)0, 0)])
 fi
 PHP_DBA_STD_RESULT(db4,Berkeley DB4)
 
